packages/react-ui-components/src/stories/Components/RelativeTimePicker.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { ComponentStory } from '@storybook/react';
+import { KvRelativeTimePicker } from '../../components';
+import React from 'react';
+
+KvRelativeTimePicker.displayName = 'KvRelativeTimePicker';
+
+export default {
+	title: 'Calendar/Time Picker/RelativeTimePicker',
+	component: 'kv-relative-time-picker',
+	argTypes: {
+		onTimezoneChange: {
+			action: 'timezoneChange'
+		},
+		onSelectedRelativeTimeChange: {
+			action: 'selectedRelativeTimeChange'
+		},
+		onCustomizeIntervalClicked: {
+			action: 'customizeIntervalClicked'
+		}
+	},
+	parameters: {
+		notes: require('@ui-notes/relative-time-picker/readme.md')
+	}
+};
+
+const RelativeTimePickerTemplate: ComponentStory<typeof KvRelativeTimePicker> = args => <KvRelativeTimePicker {...args}></KvRelativeTimePicker>;
```

Comment:
Since `KvRelativeTimePicker` doesn't have a child could we just close it` <KvRelativeTimePicker {...args}/>`

Reply:
This PR was cherry picked from the relative-time-picker branch to get everything together to launch a version. It will be updated once all the other branches are merged in dev.